### PR TITLE
Modify CompanyClient.VALID_SECURITY_FILTERS

### PR DESCRIPTION
**Pull request by MichaelJMath**
Add fields for 'is_lp', 'is_reit', and 'is_shell' to match new
    additions to API.

### CODE REVIEW

File: pycharts/clients.py [hunk 1]
```diff
@@ -5,7 +5,7 @@ class CompanyClient(BaseSecurityClient):
     
     SECURITY_TYPE_PATH = 'companies'
     VALID_SECURITY_FILTERS = ['benchmark_index', 'exchange', 'hq_region', 'incorporation_region',
-    	'industry', 'naics_industry', 'naics_sector', 'sector']
+    	'industry', 'naics_industry', 'naics_sector', 'sector', 'is_lp', 'is_reit', 'is_shell']
```
Comment (jpo424) [inline pycharts/clients.py]: @MichaelJMath, thanks for contributing! These additions are good, please keep this list in alphabetical order.

**Comment by MichaelJMath** [comment]
@jpo424 , I can pull from github. 

Regarding merging, I don't have write access to merge my own pull request, but alphabetization changes have been made.

**Comment by jpo424** [comment]
@MichaelJMath, that's right... merged!

**Comment by MichaelJMath** [comment]
Thanks for the help, @jpo424!

**Pull request merged**